crates/subspace-farmer/src/bin/subspace-farmer/commands/bench.rs

```diff
@@ -152,7 +150,15 @@ pub(crate) async fn bench(
         ),
     };
 
-    let (relay_server_node, _) = configure_relay_server(vec![]).await;
+    // Starting the relay server node.
+    let (relay_server_node, mut relay_node_runner) =
+        subspace_networking::create(Config::with_generated_keypair()).await?;
+
+    let _relay_stop_handle = tokio::spawn(async move {
```

Comment:
No need to assign handle, background task is not affected by handle drop anyway, also in other files, none of assignments there are actually necessary. Handle is typically stored when its drop does something, but this isn't the case here.